### PR TITLE
Remove unused methods in datepicker.js #6693

### LINKED ISSUE #6693: Remove unused methods in datepicker.js
The methods `getMinDateForEndDate` and `getMaxDateForStartDate` are not used in the project. There is no reason to leave them there.

### PR DESCRIPTION
**Pull request by tainansv**
Fixes #6693 

**Outline of Solution**

> Tell us how you solved the issue
I just remove the unused methods getMinDateForEndDate and getMaxDateForStartDate, as described in the issue.
Is ready for review

### CODE REVIEW

**Comment by AnaghHegde** [comment]
Can i be assigned to this?.

**Comment by wkurniawan07** [comment]
@AnaghHegde, thanks for your assistance in some of the reviews so far. However, reviews are to be done only by members who have deeper knowledge of the code base. This is a common practice in most/all projects; if you allow a newcomer who has relatively lesser knowledge of the code base to review, the review's quality is going to be compromised.
If you wish to up your skills, the best way is to work on more, harder issues. Doing one or two more difficult issues with minimum assistance has more value compared to doing/reviewing tens of issues/PRs at beginner level. The project maintainers have to do the latter not to show off our skills, but because it is our duty to make aspiring contributors feel welcome.

File: src/main/webapp/js/datepicker.js [hunk 1]
```diff
@@ -73,22 +73,6 @@ function triggerDatepickerOnClick(datepickerDivs) {
  * @assumption: startDate has a valid value
  * @returns
```
Comment (damithc) [inline src/main/webapp/js/datepicker.js]: We should remove this empty `@returns` or give more details. But that can be a separate issue. @samsontmr

Comment (samsontmr) [inline src/main/webapp/js/datepicker.js]: Now that you mention it, this is happening on line 92 as well. Shall I open a first timer issue for this?

Comment (damithc) [inline src/main/webapp/js/datepicker.js]: Yes, let's do that.

**Pull request merged**